js/arena.js

```diff
@@ -0,0 +1,60 @@
+ import Gladiator from './gladiator.js';
+
+
+export  class Arena {
```

Comment:
be sure to use export default class Arena {
to properly export the class if there is only one class in the file